src/uproot/_dask.py

```diff
@@ -1011,6 +1026,47 @@ def project_keys(self: T, keys: frozenset[str]) -> T:
         raise NotImplementedError
 
 
+def _report_failure(exception, *args, **kwargs):
+    awkward = uproot.extras.awkward()
+    return awkward.Array(
+        [
+            {
+                "duration": None,
+                "args": [repr(a) for a in args],
+                "kwargs": [[k, repr(v)] for k, v in kwargs.items()],
+                "exception": type(exception).__name__,
+                "message": str(exception),
+            }
+        ]
+    )
+
+
+def _report_success(duration, *args, **kwargs):
+    awkward = uproot.extras.awkward()
+    return awkward.Array(
+        [
+            {
+                "duration": duration,
+                "args": [repr(a) for a in args],
```

Comment:
I'm not sure I'm completely following, sorry! do we want 

```
start = time.time()
...
end = time.monotonic()
return end - start
```
?

Reply:
```python3
call_time = time.time() # since epoch, equivalent to a datetime
start = time.monotonic() # since start of program for calibration purposes
call(stuff)
stop = time.monotonic()
```